conf/evolutions/default/85.sql

```diff
@@ -0,0 +1,14 @@
+# --- !Ups
+
+ALTER TABLE `users`
+ADD COLUMN `date_created` DATETIME NULL AFTER `last_login`;
+ADD COLUMN `created_by` INT NOT NULL AFTER `date_created`;
+
```

Comment:
I swear I'm not being difficult on purpose :-/    lol

change the end of line 4 to a comma rather than a semicolon

`ALTER TABLE `users``
`ADD COLUMN `date_created` DATETIME NULL AFTER `last_login`,`
`ADD COLUMN `created_by` INT NOT NULL AFTER `date_created`;`